packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6690,14 +6707,16 @@ export default class Meeting extends StatelessWebexPlugin {
 
   /**
    * Handle logging the media
-   * @param {Object} audioTrack The audio track
-   * @param {Object} videoTrack The video track
+   * @param {Object} mediaProperties
    * @private
    * @returns {undefined}
    */
-  private handleMediaLogging({audioTrack, videoTrack}: any) {
-    MeetingUtil.handleVideoLogging(videoTrack);
-    MeetingUtil.handleAudioLogging(audioTrack);
+  private handleMediaLogging(mediaProperties: {
+    audioTrack: LocalMicrophoneTrack | null;
```

Comment:
I've noticed a lot of `| null` any reason for it?
Why not make the params optional?

Reply:
Yeah I agree it would be better as optional. I used the null, because there was already many places where these tracks were being set to null and wanted to minimize the size of this PR - and it's not always obvious, see for example [here](https://github.com/webex/webex-js-sdk/blob/beta/packages/%40webex/plugin-meetings/src/meeting/index.ts#L5812). It will be easier to clean this up further once we remove the existing transcoded methods that use streams so I'd rather do it as part of SPARK-399695